src/eip7594/eip7594.c

```diff
@@ -454,6 +454,125 @@ static C_KZG_RET compute_r_powers_for_verify_cell_kzg_proof_batch(
     return ret;
 }
 
+/**
+ * Aggregate columns, compute the sum of interpolation polynomials, and commit to the result.
+ *
+ * This function computes: [sum_k r^k interpolation_poly_k(s)]
```

Comment:
What is `interpolation_poly` exactly? Should this be?
```suggestion
 * This function computes: [sum_k r^k aggregated_interpolation_poly_k(s)]
```

Reply:
Clarified that this is a quote from the spec. It might help people who read the spec and match it with the code.